### DISCUSSION TITLE
Cut and paste

### DISCUSSION THREAD

**Discussion by jsismail** [vincode-io/Zavala#238] [Support] [answered]
Is there a way to cut and paste portions of an outline, between outlines?  I can’t seem to find a way to select multiple lines..

**Comment by vincode-io** [comment, marked as answer]
Yes. You can use the regular cut and paste functions. To select multiple lines you hold down either the shift or command key while clicking rows.

**Reply by jsismail** [reply]
Thanks!  Holding down the command key does the trick.
